external/ag-website-shared/src/content/license-features/chartsFeaturesMatrix.json

```diff
@@ -307,17 +307,17 @@
             },
             {
                 "label": {
-                    "name": "Annotations",
-                    "link": "https://ag-grid.com/charts/react/annotations/"
+                    "name": "Toolbar",
```

Comment:
is annotations no longer an option?

Reply:
The link doesn't seem to be available anymore. Seems like it moved to https://ag-grid.com/charts/react/financial-charts-toolbar/#annotations, but I'll check with David

Reply:
Yep, it's as it should be